src/scripts/components/main/onboarding/onboarding.tsx

```diff
@@ -0,0 +1,147 @@
+import React, {FunctionComponent, useEffect, useState} from 'react';
+import {useIntl} from 'react-intl';
+import cx from 'classnames';
+
+import Button from '../button/button';
+import {CloseIcon} from '../icons/close-icon';
+
+import styles from './onboarding.styl';
+
+interface Props {
+  id: number;
+  onPageChange: (id: number) => void;
+  onClose: () => void;
+}
+
+const Onboarding: FunctionComponent<Props> = ({id, onPageChange, onClose}) => {
+  const intl = useIntl();
+
+  const onboardingContent = [
+    {
+      id: 1,
```

Comment:
If the id is just the index it's probably better to just use the index and call the prop `step` or something. No need to manually fill in the ids here.